test/test_jit.py

```diff
@@ -6839,6 +6876,9 @@ def clone_inputs(requires_grad):
     outputs_test = func(*nograd_inputs, **kwargs)
     self.assertEqual(outputs, outputs_test)
 
+    graph, graph_inputs = func_graph_inputs(*nograd_inputs, **kwargs)
+    self.assertExportImport(graph, graph_inputs)
```

Comment:
This seems like it only does a subset of the tests on an imported and exported graph. For instance, it skips all the grad tests below. 

Furthermore, it causes weird duplication in the all of the harness code so that you can call `assertExportImport`. 

It seems like it would be easier to just call `getExportImportCopy(m)` inside of `create_traced_fn` and `create_script_fn` and have them accept flags for whether to import/export or not.

Reply:
I though there was some value in separating out the export / import tests from the creation of traced_fn/script_fn. You're right that the code is much simpler. I have updated to the way you suggested.